src/operator/nn/dropout-inl.h

```diff
@@ -337,6 +336,7 @@ class DropoutOp {
   real_t pkeep_;
   /*! \brief Dropout mode */
   dropout::DropoutOpMode mode_;
+  TShape axes;
```

Comment:
nit: member variable name should end in an underscore

Reply:
Thx 👍